lib/asciidoctor-pdf/converter.rb

```diff
@@ -296,7 +296,7 @@ def build_pdf_info doc
     info[:Creator] = str2pdfval %(Asciidoctor PDF #{::Asciidoctor::Pdf::VERSION}, based on Prawn #{::Prawn::VERSION})
     info[:Producer] ||= (info[:Author] || info[:Creator])
     # FIXME use docdate attribute
-    info[:ModDate] = info[:CreationDate] = ::Time.now
+    info[:ModDate] = info[:CreationDate] = ::Time.now if doc.attr 'timestamps'
```

Comment:
I wasn't clear if doing "timestamps!" would result in the value being absent altogether. If so, that's a fine solution and I'm happy to make the change. Otherwise, if it's still present but false, I think that will break functionality.